docs/deployment.md

```diff
@@ -21,6 +21,8 @@ The ASGI application should be specified in the form `path.to.module:instance.pa
 
 When running locally, use `--reload` to turn on auto-reloading.
 
+Keep in mind, that `--reload` disables `--workers`.
```

Comment:
what about something like, "the reload and workers flag are mutually exclusive, use one or the others" ?
the intent is that reload is aimed at development and workers at production, even if I'm not sure why you'd wanna use it but that's another topic.

Reply:
Yes, makes sense. I can update this.
Background when/why to use both options are in #1413 
(in short: calling the "own" API (via "requests" call, not method calls) blocks the system when not using multiple workers. That's at least my understanding... during development...)

Reply:
both flags use multiprocess under the hood

Reply:
I've added a comment to further discuss the issue here: https://github.com/encode/uvicorn/issues/1413#issuecomment-1074948326

Reply:
not sure if there is a misunderstanding here, this PR is good, just need to rephrase it a bit

Reply:
Hi @euri10 
sorry for the delay... I pushed to changes. Do you think this is ready?